ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/requesthandler/sendtransaction/TransactionTransmitter.java

```diff
@@ -62,21 +63,25 @@ public TransactionTransmitter(
   }
 
   public void send() {
-    createSignedTransactionBody();
+    final Optional<JsonRpcRequest> request = createSignedTransactionBody();
+
+    if (request.isEmpty()) {
+      return;
+    }
+
+    try {
+      sendTransaction(Json.encodeToBuffer(request.get()));
+    } catch (final IllegalArgumentException e) {
```

Comment:
Should we also catch the EncodeException that encodeToBuffer throws?

Reply:
Yes - done